extensions/windows-event-log/wel/MetadataWalker.cpp

```diff
@@ -17,9 +17,15 @@
  */
 
 #include <windows.h>
+#include <strsafe.h>
+
+#include <map>
+#include <string>
+#include <utility>
+#include <vector>
```

Comment:
Missing: `<functional>`, `<codecvt>`, `<regex>`,

Reply:
thanks, fixed in [95aeacd](https://github.com/martinzink/nifi-minifi-cpp/commit/95aeacd78262361de87609058f1f208b01e4fe3b)